src/preamble.js

```diff
@@ -930,11 +930,11 @@ function stackTrace() {
 
 // Memory management
 
-var PAGE_SIZE = 4096;
+var PAGE_SIZE = 65536; // Wasm pages are fixed to 64k.
```

Comment:
Oh, sorry about that; I did mean 64k instead of 16k. It should be 64k everywhere, so `PAGE_SIZE` should match the increments of `grow_memory`. And in the future if/when there are other memory primitives in wasm they will be defined in increments of 64k too.